inspire_json_merger/config.py

```diff
@@ -321,3 +322,4 @@ class GrobidOnArxivAuthorsOperations(MergerConfigurationOperations):
         'authors.raw_affiliations': U.KEEP_ONLY_UPDATE_ENTITIES
     }
     comparators = GROBID_ON_ARXIV_COMPARATORS
+    conflict_types_to_delete = ['SET_FIELD']
```

Comment:
sorry if I was unclear, this is not what I meant at all. What I meant is to add `conflict_filters = ["authors.full_name"]` to the GROBID config.